lib/utils/dom.js

```diff
@@ -343,3 +343,16 @@ function addStyle(css) {
 
 	return style;
 }
+
+export function dataAttributes(element: HTMLElement): any {
```

Comment:
Can [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) be used instead?